lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -282,6 +282,30 @@ GetPointerTo(swift::Demangle::Demangler &dem,
   return bgs;
 }
 
+/// Builds a bound generic struct demangle tree with the name, module name, and
+/// the struct's elements.
+static swift::Demangle::NodePointer GetBoundGenericStruct(
```

Comment:
The naming convention we use for these helpers is CreateBoundGenericStruct(). (There is also CreateTupleType, CreateGenericTypeParamType, CreateSILPackType, ...)